client/js/screens/Campaigns/styles.js

```diff
@@ -1,3 +1,88 @@
 import {StyleSheet} from 'react-native';
-const styles = StyleSheet.create({});
+const styles = StyleSheet.create({
+  xxxIcon: {
+    resizeMode: 'contain',
+    height: 30,
+    alignSelf: 'flex-end',
+  },
+  categoryImage: {
+    height: 60,
+    width: 100,
+    resizeMode: 'contain',
+    // alignSelf: 'flex-start',
+  },
+  campaignsHeader: {
+    justifyContent: 'space-between',
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingTop: 10,
+    marginBottom: 10,
+    marginRight: 60,
+    marginLeft: 20,
```

Comment:
Please use flexbox properties instead of huge margins.

Reply:
Hey I showed you this screen when I created it. The margin is the only way to do this because the image isn't exactly center. There is already flex applied to the above images.
You said it was alright.